### PR TITLE
Adding badges

### PR DESCRIPTION
**Pull request by andrewrlee**
Adding the circle-ci and moj-compliant badges and adjusting the README so new projects will inherit these by default

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,6 @@
     "npm": "^8"
   },
   "jest": {
-    "preset": "ts-jest",
```
Comment (andrewrlee) [inline package.json:36]: don't think this is needed any more?

Comment (psoleckimoj) [reply package.json:36]: It's a base config? We are pointing at the `ts-jest` module? https://jestjs.io/docs/configuration#preset-string

**Review by psoleckimoj — DISMISSED** [review]
LGTM!

File: README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # hmpps-template-typescript
+[![repo standards badge](https://img.shields.io/badge/dynamic/json?color=blue&style=for-the-badge&logo=github&label=MoJ%20Compliant&query=%24.result&url=https%3A%2F%2Foperations-engineering-reports.cloud-platform.service.justice.gov.uk%2Fapi%2Fv1%2Fcompliant_public_repositories%2Fhmpps-template-typescript)](https://operations-engineering-reports.cloud-platform.service.justice.gov.uk/public-github-repositories.html#hmpps-template-typescript "Link to report")
```
Comment (neilmendum) [inline README.md:2]: Not a big thing but in https://github.com/ministryofjustice/hmpps-incentives-api/pull/63 I used `style=flat` so the badge was the same size & shape as others

Comment (andrewrlee) [reply README.md]: ah that looks better! @neilmendum are you ok to merge this if you're happy with it?

**Review by neilmendum — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (andrewrlee) [reply package.json:36]: I spent ages reading the docs and I still don't know. The tests run successfully without it

**Pull request merged**